src/Server/SSHPipeInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/* (c) Anton Medvedev <anton@medv.io>, Maxim Kuznetsov <skypluseg@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Deployer\Server;
+
+interface SSHPipeInterface
```

Comment:
What's strange for you? I think that there is a chance in the future that we need to add new type of `Server`, so we need to define which of them supports `ssh` task and which one does not. If we have an interface then we can be sure that a class implemented one has concrete method like `createSshPipe()`. Or do you think that checking `getenv('ssh_type')` is enough?